owncloudComLibrary/build.gradle

```diff
@@ -16,11 +16,11 @@ allOpen {
 }
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
```

Comment:
Updating Kotlin and Gradle is fine but targetting Android Q is something we tackle carefully since the something could break. We will target Q in the app and the library from a separate issue https://github.com/owncloud/android/issues/2576 , so could you keep 28 here?

Reply:
I reverted target to 28